BuyPolish/PolaTestsData/krasnystaw.json

```diff
@@ -0,0 +1 @@
+{"product_id": 9002, "code": "5902057000475", "name": "OKR\u0118GOWA SP\u00d3\u0141DZIELNIA MLECZARSKA W KRASNYMSTAWIE", "card_type": "type_white", "plScore": 100, "altText": null, "plCapital": 100, "plCapital_notes": "Sp\u00f3\u0142ka rodzima. \r\n\r\nhttp://krasnystaw.eu/?page_id=12", "plWorkers": 100, "plWorkers_notes": "Sp\u00f3\u0142ka produkuje swoje produkty na terenie Polski, w Krasnymstawie. \r\n\r\nhttp://krasnystaw.eu/?page_id=23", "plRnD": 100, "plRnD_notes": "Sp\u00f3\u0142ka posiada na miejsca pracy w obszarze BiR ulokowane na terenie Polski.\r\n\r\nhttp://krasnystaw.eu/?page_id=23\r\nhttp://krasnystaw.eu/?page_id=12\r\nhttp://krasnystaw.eu/", "plRegistered": 100, "plRegistered_notes": "Sp\u00f3\u0142ka zarejestrowana na terenie Polski.\r\n\r\nhttp://www.krs-online.com.pl/okregowa-spoldzielnia-mleczarska-w-krs-102714.html", "plNotGlobEnt": 100, "plNotGlobEnt_notes": "Firma polska.\r\n\r\nhttp://krasnystaw.eu/?page_id=12", "report_text": "Zg\u0142o\u015b je\u015bli posiadasz bardziej aktualne dane na temat tego produktu", "report_button_text": "Zg\u0142o\u015b", "report_button_type": "type_white", "is_friend": false, "description": "Krasnystaw OSM to nowoczesna, polska firma wsp\u00f3\u0142pracuj\u0105ca z ponad trzystoma dystrybutorami produkt\u00f3w nabia\u0142owych z ca\u0142ej Polski. Jej tradycja si\u0119ga pocz\u0105tk\u00f3w minionego wieku \u2013 zosta\u0142a bowiem za\u0142o\u017cona w 1913 roku. Obecnie odbiera mleko od ponad 2000 dostawc\u00f3w, przetwarza rocznie 160 mln litr\u00f3w mleka", "sources": {"Dane z KRS": "http://mojepanstwo.pl/dane/krs_podmioty/107925", "Strona producenta": "http://krasnystaw.eu/?page_id=12"}, "ai": {"ask_for_pics": true, "ask_for_pics_preview": "Pola Ci\u0119 potrzebuje", "ask_for_pics_title": "Naucz Pol\u0119 tego produktu", "ask_for_pics_text": "Pom\u00f3\u017c nauczy\u0107 Pol\u0119 rozpoznawania produkt\u00f3w po wygl\u0105dzie. Nakr\u0119\u0107 film obracaj\u0105c produkt tak jak na przyk\u0142adzie poni\u017cej. Postaraj si\u0119 nie zas\u0142ania\u0107 produktu.", "ask_for_pics_product": "Uczysz Pol\u0119 produktu: Krasnystaw \u015amietana 18% homogenizowana 360 g", "ask_for_pics_button_start": "Nakr\u0119\u0107 film", "max_pic_size": 800}, "donate": {"show_button": true, "url": "https://klubjagiellonski.pl/zbiorka/wspieraj-aplikacje-pola/", "title": "Wspieraj aplikacj\u0119 Pola"}}
```

Comment:
Jakbyś przeciągnął to przez https://jsonformatter.org/json-pretty-print byłoby o wiele łatwiej czytam :) Jakbyś mógł przy okazji to samo zrobić z lomza.json to w ogóle byłoby ekstra.